chat/message.go

```diff
@@ -3,10 +3,33 @@ package chat
 import (
 	"bytes"
 	"fmt"
+	"time"
 
 	"github.com/golang/protobuf/jsonpb"
+	timestamp "github.com/golang/protobuf/ptypes/timestamp"
 )
 
+/*
+Create constructs a new `Message` with a default timestamp of now */
```

Comment:
exported messages should look like this I believe:

`CreateMessage constructs ...`

(you need to use the name of the funciton as the first word)

VS code golang's functionality makes me do this or it won't compile :/

Reply:
Good catch, ill update